src/templates/components/affiliate-multiple-event-component-venue.njk

```diff
@@ -0,0 +1,39 @@
+<!-- MULTIPLE EVENTS -->
+<table role="presentation" align="center" border="0" cellpadding="0" cellspacing="0" class="emailwrapto100pc" width="600">
+    <tbody>
+        <tr>
+            <td style="font-family:  Helvetica, Arial, sans-serif; font-size:16px; color:#2a2a2a;" bgcolor="#f2f2f2">
+                <table role="presentation" bgcolor="#b2cce5" width="100%" border="1" cellspacing="0" cellpadding="0" style="background-color:#b2cce5; border-radius: 3px; border-collapse: separate;" align="right" class="footer-right">
+                    <tbody>
+                        <tr>
+                            <th align="center" style="padding: 8px 0 0 0;">Event</th>
+                            <th align="center" style="padding: 8px 0 0 0;">Date</th>
+                            <th align="center" style="padding: 8px 0 0 0;">Location</th>
+                        </tr>
+                        <tr>
+                            <td align="center"  style="padding: 5px;"><a href="">Trans Resistance Festival</a></td>
+                            <td align="center"  style="padding: 5px;"><a href="">Sat, June 3,<br /> 11 a.m. &ndash; 5 p.m.</a></td>
+                            <td align="center"  style="padding: 5px;"><a href="">Stony Brook Station</a></td>
+                        </tr>
+                        <tr>
+                            <td align="center"  style="padding: 5px;"><a href="">Boston Pride for the People Parade</a></td>
+                            <td align="center"  style="padding: 5px;"><a href="">Sat, June 10,<br /> 10 a.m. &ndash; 8 p.m.</a></td>
+                            <td align="center"  style="padding: 5px;"><a href="">Copley Square</a></td>
+                        </tr>
+                        <tr>
+                            <td align="center"  style="padding: 5px;"><a href="">Action Team Media Crew Training</a></td>
+                            <td align="center"  style="padding: 5px;"><a href="">Wed, Aug 16,<br /> 5:30 &ndash; 7:30 p.m.</a></td>
```

Comment:
this is just a formatting thing and not code, but abbreviated day and months should have a period at the end. Example `Wed., Aug. 16`